api/nucleus/views.py

```diff
@@ -1,16 +1,22 @@
-from django.shortcuts import render
+from django.conf import settings
+from django.shortcuts import redirect, render
 from social_django.utils import psa, load_strategy
 
 
 def require_email(request):
-    strategy = load_strategy()
     partial_token = request.GET.get('partial_token')
-    partial = strategy.partial_load(partial_token)
-    context = {
-        'email_required': True,
-        'partial_backend_name': partial.backend,
-        'partial_token': partial_token
-    }
-    return render(request, 'nucleus/require_email.html', context)
+    return_url = '{protocol}://{domain}/finish-steam/{token}'.format(
+        protocol='http' if settings.DEBUG else 'https',
+        domain='127.0.0.1:3000',
+        token=partial_token
+    )
+    return redirect(return_url)
 
 
+def social_redirect(request):
```

Comment:
This is the view that redirects users who are signing in (already have an account).